test/src/test_gehrd.cpp

```diff
@@ -129,7 +129,8 @@ TEMPLATE_LIST_TEST_CASE("Hessenberg reduction is backward stable", "[eigenvalues
     DYNAMIC_SECTION("GEHRD with"
                     << " matrix = " << matrix_type << " n = " << n << " ilo = " << ilo << " ihi = " << ihi << " nb = " << nb)
     {
-        gehrd_opts_t<idx_t, T> opts = {.nb = nb, .nx_switch = 2};
+        gehrd_opts_t<idx_t, T> opts;
```

Comment:
This is necessary because `gehrd_opts_t<idx_t, T> opts = {.nb = nb, .nx_switch = 2};` is only available on C++20.